### PR TITLE
Add RW+Grafana tutorial

### PR DESCRIPTION
**Pull request by emile-00**
Add "Using RisingWave to monitor RisingWave metrics" tutorial.

## Info
- **Description**: 
[ What's changed? ]

- **Related code PR**: 
[ Provide the link to the code PR here. For example, https://github.com/risingwavelabs/risingwave/pull/4085. Delete this part if there's no code PR related. ]

- **Related doc issue**: 
Resolves [ Provide the link to the doc issue here. ]

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Review by CharlieSYH — APPROVED** [review]
May need to change the "Using" in the title to "Use".
See the style guide: https://www.notion.so/risingwave-labs/Headings-and-titles-d044099e246e4ec19f182d97143b6fdd

File: docs/tutorials/monitor-rw-metrics.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+}
+```
+
+Once the Kafka topic is ready, we can prepare the RisingWave SQL pipeline. The following SQL statement registers the Kafka topic to RisingWave.
+
+```sql
+CREATE SOURCE prometheus (
+    labels STRUCT < __name__ VARCHAR,
+    instance VARCHAR,
+    job VARCHAR >,
```
Comment (fuyufjh) [inline docs/tutorials/monitor-rw-metrics.md]:
```suggestion
    labels STRUCT <
        __name__ VARCHAR,
        instance VARCHAR,
        job VARCHAR
    >,
```

**Review by fuyufjh — APPROVED** [review]
LGTM!

**Review by neverchanje — APPROVED** [review]
Exellent work! LGTM

File: docs/tutorials/monitor-rw-metrics.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
+---
+id: using-risingwave-to-monitor-risingwave-metrics
+slug: /using-risingwave-to-monitor-risingwave-metrics
+title: Using RisingWave to monitor RisingWave metrics
+description: Use RisingWave to monitor Risingwave metrics.
+---
+
+## Overview
+
+RisingWave uses Prometheus to collect the system metrics. Prometheus is a powerful monitoring platform that provides an end-to-end solution from instrumenting applications to querying metrics. However, Prometheus’s local storage is limited to single-node durability and scalability. To replicate the data from local storage to remote storage systems, we can use a proxy service that sends data in JSON format to Kafka. Then RisingWave can read, store, and perform complex queries on the data from Kafka.
```
Comment (hengm3467) [inline docs/tutorials/monitor-rw-metrics.md]: To replicate the data -> To replicate data

File: docs/tutorials/monitor-rw-metrics.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+```
+
+Now navigate to the `prometheus` directory and start the demo cluster from the docker compose file.
+
+```shell
+cd risingwave-demo/prometheus
+docker-compose up -d
+```
+
+Necessary RisingWave components, Prometheus, and Grafana will be started.  Prometheus will start collecting data from RisingWave and write them to Kafka topics. In this demo cluster, the data ingested from the Kafka topic will be stored in the MinIO instance.
```
Comment (hengm3467) [inline docs/tutorials/monitor-rw-metrics.md]: There is an addition space after the first sentence in the last paragraph.

**Review by hengm3467 — APPROVED** [review]

**Pull request merged**